pom.xml

```diff
@@ -1545,7 +1545,7 @@
 
         <carbon.analytics.common.version>5.2.50</carbon.analytics.common.version>
         <carbon.analytics.common.imp.pkg.version>[5.2.0, 6.0.0)</carbon.analytics.common.imp.pkg.version>
-        <fasterxml.jackson.version>2.13.3</fasterxml.jackson.version>
+        <fasterxml.jackson.version>2.14.1</fasterxml.jackson.version>
```

Comment:
In MI 4.1.0, we couldn't upgrade to this version due to some test failures. Could you please verify it? Refer https://github.com/wso2/micro-integrator/pull/2829

Reply:
Upgraded to latest 2.13.x version instead with https://github.com/wso2/micro-integrator/pull/2838/commits/5a91eabe15074eecf878f7a25db0f72b2609db4e